collect_app/src/test/java/org/odk/collect/android/preferences/GeneralSharedPreferencesTest.java

```diff
@@ -40,6 +40,7 @@ public void save_allowsNullValues() {
     }
 
     @Test(expected = GeneralSharedPreferences.ValidationException.class)
+    @SuppressWarnings("PMD.UseAssertTrueInsteadOfAssertEquals")
```

Comment:
Generally agree with the check here but feel like an `assertFalse` would make the code less readable (and considerable) as we're asserting on the value stored rather than asserting on a result or flag.